File: core/src/commonMain/kotlin/work/socialhub/kmastodon/api/PollResource.java
package work.socialhub.kmastodon.api;

import mastodon4j.entity.Poll;
import mastodon4j.entity.share.Response;

/**
 * @author uakihir0
 */
public interface PollResource {

    /**
     * Vote on a poll.
     */
    public Response<Poll> votePoll(String id, long[] choices);
}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/api/PollResource.kt
package work.socialhub.kmastodon.api

import mastodon4j.entity.Poll
import mastodon4j.entity.share.Response

/**
 * @author uakihir0
 */
interface PollResource {
    /**
     * Vote on a poll.
     */
    fun votePoll(id: String?, choices: LongArray?): Response<Poll?>?
}
